docs/hazmat/primitives/asymmetric/index.rst

```diff
@@ -32,4 +32,36 @@ and Elliptic Curve.
     serialization
     utils
 
+.. module:: cryptography.hazmat.primitives.asymmetric
+
```

Comment:
We could create a docs/hazmat/primitives/asymmetric/interfaces.rst and link it in the toc but define its module as `cryptography.hazmat.primitives.asymmetric` so the module path links work properly.